submissions/Eugene-CG/a-tiny-JS-world/index.js

```diff
@@ -0,0 +1,81 @@
+/* Refer to https://github.com/OleksiyRudenko/a-tiny-JS-world for the task details
+   Complete the below for code reviewers' convenience:
+
+   Code repository: https://github.com/Eugene-CG/a-tiny-JS-world
+   Web app: https://eugene-cg.github.io/a-tiny-JS-world/
+   */
+
+// ======== OBJECTS DEFINITIONS ========
+// Define your objects here
+
+const cat = {
+  species: "cat",
+  name: "Eugene",
+  gender: "male",
+  legs: 4,
+  hands: 0,
+  saying: "Hey Fellas!",
+};
+const dog = {
+  species: "dog",
+  name: "Toby",
+  gender: "male",
+  legs: 4,
+  hands: 0,
+  saying: "woof-woof!",
+};
+const man = {
+  species: "man",
+  name: "John",
+  gender: "male",
+  legs: 2,
+  hands: 2,
+  saying: "I am a man",
+};
+const woman = {
+  species: "dog",
+  name: "Yennefer",
+  gender: "female",
+  legs: 2,
+  hands: 2,
+  saying: "Nice to meet you",
+};
+const catWoman = {
+  species: "Dream",
+  name: "Noone",
+  gender: "female",
+  legs: 2,
+  hands: 2,
+  saying: cat.saying,
+};
+const arr = [cat, dog, man, woman, catWoman];
+let string = [];
```

Comment:
Same thing, but even more confusing. A `string` that is actually and array? Also, you could've `print`ed every inhabitants info separately, it is not against the task, but since you are already doing this, why `let`? You are not replacing the value anywhere, you are using `push`. It is allowed to change the value of arrays with functions like `push` even when the variable is a `const`.